### DISCUSSION TITLE
Windows 11 - Left Click on Start Menu not working

### DISCUSSION THREAD

**Discussion by Harry-The-Greek** [valinet/ExplorerPatcher#1377] [General]
Hello Everyone ,

Firstly, let me congratulate Valentin-Gabriel Radu, for creating this great application.
Dude THANK you VERY MUCH for this program ! 
You really held us all, while Microsoft, really blue it on the Start Menu.

Now about my problem.
I am having a problem when i left click on Start button and i should have the Options appearing.
I am using the Version 22621.1555.55.1 and Windows 11 ver. 22H2 (OS Build 22621.1631)

I uninstalled the ExplorerPatcher and i got back the left click option on the Start Menu, without even restarting.
Then i re-installed the latest version of ExplorerPatcher but the left click is not working.
I restored back to default settings --> About --> Restore default Settings, rebooted the computer but the problem still exists.

Does anyone knows anything i could try doing, in order to get back the left click, while using the ExplorerPatcher program?
Something went wrong, somewhere, and lost the menu when i left click on the Start Button.

Thank you for your time,
Harry.

**Comment by TheAncient2** [comment]
Same problem here, except with RIGHT-click. (Running EP 22621.1555.55.**2** under Windows 11 ver. 22H2 (OS Build 22621.163**5**))

**Comment by Harry-The-Greek** [comment]
Ok.
I have installed the latest version 22621.1555.55.2.
I went to Updates --> Clicked on the Receive pre-release versions, if available 
and after the installation the left click worked again.

Thank you !!

P.S. I used the RegSeeker 4.7.364018 as Administrator and Search the registry for anything like ExplorerPatcher.
I deleted everything i found in the registry and then reinstalled the 22621.1555.55.1 version.
I had the same issues, so I when to **_Updates_** --> Clicked on the **_Receive pre-release versions, if available_** 
and then i choose to **_Update the program and restart File Explorer_**

**Reply by faust6669** [reply]
Man, I have been trying for a few days to get my start menu to work, getting pre release versions is the only thing that worked, thank you.

**Comment by theking2** [comment]
Is it true that the start menu stops appearing _only_ when the taskbar is moved to the top?

**Reply by TheAncient2** [reply]
When this happened to me, I had the taskbar at the bottom.
The problem has since been resolved (running 22621.1555.55.2 now)
I did NOT go through all the gyrations that Harry-The-Greek went through i.e. cleaning out the registry. I simply installed the latest version.
